src/commonMain/kotlin/character/classes/priest/specs/Smite.kt

```diff
@@ -0,0 +1,25 @@
+package character.classes.priest.specs
+
+import character.Spec
+import character.SpecEpDelta
+
+class Smite : Spec() {
+    override val name: String = "Smite"
+    override val epBaseStat: SpecEpDelta = spellPowerBase
+    override val epStatDeltas: List<SpecEpDelta> = defaultCasterDeltas
+
+    override fun redSocketEp(deltas: Map<String, Double>): Double {
+        // 9 spell dmg
+        return 9.0
+    }
+
+    override fun yellowSocketEp(deltas: Map<String, Double>): Double {
+        // 4 spell hit + 5 spell dmg
+        return 11.16
```

Comment:
This should use the computed delta arg value of spell hit instead of a fixed number:

`return ((deltas["spellHitRating"] ?: 0.0) * 5.0) + 5`